lib/Models/CatalogGroup.js

```diff
@@ -346,6 +315,8 @@ CatalogGroup.prototype._getValuesThatInfluenceLoad = function() {
  * @param {CatalogMember} item The item to add.
  */
 CatalogGroup.prototype.add = function(item) {
+    item.parent = this;
```

Comment:
This `parent` property should be defined in the `CatalogMember` constructor, rather than added dynamically later.  It can be undefined initially, of course.

Currently CkanCatalogGroup can and does add a single `CatalogItem` to multiple `CatalogGroups`.  :fearful: 

We could potentially change that, or maybe there's another way to do this?